tools/research_agent.py

```
@@ -0,0 +1,369 @@
+    generate_parser.add_argument(
+        "--template",
+        "-t",
+        default="executive_summary",
+        help="Template key to use. Options: {}".format(", ".join(sorted(TEMPLATES))),
+    )
+    generate_parser.add_argument("--stakeholder", help="Intended audience to emphasize in the report.")
+    generate_parser.add_argument("--researcher", help="Name or team preparing the dossier.")
+    generate_parser.add_argument("--notes", help="Extra context recorded in metadata.")
+    generate_parser.add_argument("--output", "-o", help="Optional path to write the dossier Markdown file.")
+
+    list_parser = subparsers.add_parser("list-templates", help="Show available report templates.")
```

Ruff (0.14.1)

Local variable `list_parser` is assigned to but never used

Remove assignment to unused variable `list_parser`

(F841)